src/controllers/queries/index.ts

```diff
@@ -184,6 +202,116 @@ export class QueriesController extends HTMLController {
     )
   }
 
+  /**
+   * Retrieves the query response page
+   */
+  @SuccessResponse(200)
+  @Get('/scope-3-carbon-consumption-response/{queryId}')
```

Comment:
I think we should clean up the path a bit here. Maybe something like `/scope-3-carbon-consumption/{queryId}/response`